index.html

```diff
@@ -4,5 +4,9 @@
     overflow: hidden;
   }
 </style>
-<canvas />
+<div style="width: 100%; height: 100%; background-color: #0e0e0e">
+  <div style="width: 1280; height: 720; margin: auto">
```

Comment:
Why not just apply this directly on the canvas itself?

Reply:
Well I didn't see our actual HTML tag, so I figured I'd just throw in a div to get our background color for the whole page